ui/imports/shared/views/SendModalHeader.qml

```diff
@@ -85,12 +85,12 @@ StatusFloatingButtonsSelector {
         implicitWidth: 272
         title: name
         subTitle: currencyBalance
-        icon.emoji: !!emoji ? emoji: ""
-        icon.color: model.color
-        icon.name: !emoji ? "filled-account": ""
-        icon.letterSize: 14
-        icon.isLetterIdenticon: !!model.emoji
-        icon.background.color: Theme.palette.indirectColor1
+        asset.emoji: !!emoji ? emoji: ""
+        asset.color: model.color
+        asset.name: !emoji ? "filled-account": ""
+        asset.letterSize: 14
+        asset.isLetterIdenticon: !!model.emoji
```

Comment:
can isImage and isLetterIdenticon be made into a single Enum? making it mutually exclusive? this maybe confusing to use?

Reply:
might be. but also might be that none is needed, if we internally check whether the requested icon name exists in icons, if not then it's an image.. however this requires more work, would it be ok to create a new issue?

Reply:
sure :)